packages/dependency-extraction-webpack-plugin/test/build.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const glob = require( 'glob' ).sync;
+const mkdirp = require( 'mkdirp' ).sync;
+const path = require( 'path' );
+const rimraf = require( 'rimraf' ).sync;
+const webpack = require( 'webpack' );
+
+/**
+ * Internal dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '..' );
+
+const fixturesPath = path.join( __dirname, 'fixtures' );
+const configFixtures = fs.readdirSync( fixturesPath ).sort();
+
+afterAll( () => rimraf( path.join( __dirname, 'build' ) ) );
+
+configFixtures.forEach( ( configCase ) => {
```

Comment:
There is also `describe.each` which plays a similar role:
https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout